src/Yaapii.Http/Parts/Bodies/TextBody.Of.cs

```diff
@@ -46,6 +47,11 @@ public sealed class Of : TextEnvelope
             /// </summary>
             public Of(IDictionary<string, string> input) : base(() =>
                 {
+                    new FailWhen(
```

Comment:
You could use the FallbackMap from Atoms - it would be more OO because it is an object which validates while using it.